MatrixSDK/Data/MXRoomLastMessage.m

```diff
@@ -58,6 +59,22 @@ - (instancetype)initWithEvent:(MXEvent *)event
     return self;
 }
 
+- (nullable NSData*)sensitiveData;
+{
+    NSData* archived = [NSKeyedArchiver archivedDataWithRootObject:[self sensitiveDataDictionary]
+                                             requiringSecureCoding:NO
+                                                             error:nil];
```

Comment:
Is it worth logging any potential archiving errors that occur?

Reply:
Archiving a dictionary is quite safe, but added it anyway!
https://github.com/matrix-org/matrix-ios-sdk/pull/1718/commits/e6a0aee351bf4d30363767b5c7c81cdebfdbfa09